di_art/test/parameter_test.dart

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
I would also suggest to place tests in the same folders as we have packages

Reply:
Do you mean structure like this:
test->paramater->parameter_test.dart?

Reply:
> Do you mean structure like this:
> test->paramater->parameter_test.dart?

If you will change structure change only `parameter`. I already change other folders.

Reply:
Roger it :)